src/meshapi/views/forms.py

```diff
@@ -117,24 +133,47 @@ def join_form(request: Request) -> Response:
             {"detail": "Your address could not be validated."}, status=status.HTTP_500_INTERNAL_SERVER_ERROR
         )
 
-    # Check if there's an existing member. Dedupe on email for now.
-    # A member can have multiple install requests
-    existing_members = Member.objects.filter(
-        Q(primary_email_address=r.email)
-        | Q(stripe_email_address=r.email)
-        | Q(additional_email_addresses__contains=[r.email])
+    # Check if there's an existing member. Group members by matching on both email and phone
```

Comment:
Idea, could we gate creating new members in the admin panel with this logic?

Reply:
Maybe also checking that data (email, phone, etc) you enter doesn't already belong to another Member.

Reply:
Added an issue to track this request https://github.com/nycmeshnet/meshdb/issues/463